specification/sections/01-flag-evaluation.md

```diff
@@ -4,15 +4,15 @@ description: The specification that defines the developer facing feature flag ev
 toc_max_heading_level: 4
 ---
 
-# Flag Evaluation API
+# 1. Flag Evaluation API
```

Comment:
Is a number required here? If so, can it just be `1` instead of `1.`?

Reply:
I think everything we apply a level to needs a number.

`1 Flag Evaluation API` looked very weird to me, and most section numbers in a sectioned document end with a `.` see https://datatracker.ietf.org/doc/html/rfc6749#section-3 for example.